lib/IRGen/IRGenModule.h

```diff
@@ -1057,10 +1057,10 @@ class IRGenModule {
                               SILType objectType, const TypeInfo &objectTI,
                               const OutliningMetadataCollector &collector);
 
-private:
```

Comment:
IIRC there was a review comment here saying I should use a higher level of abstraction. (Just leaving this comment so I remember.)

Reply:
Actually, I think this is the correct method to use. I'm not sure what the problem with making it public is. If there is a more appropriate API, let me know and I'll use that. I suppose I could just use `IGM.ClangCodeGen->GetAddrOfGlobal` directly, but that seems worse to me.